doc/Caliptra.md

```diff
@@ -943,7 +945,7 @@ Caliptra provides a HW API to do a SHA384 hash calculation. The SoC can access t
 
 ### Architectural registers
 
-These registers are accessible over APB to be read according to the register access permissions. For more information, see the register reference manual at https://ereg.caliptra.org.
+These registers are accessible over APB to be read according to the register access permissions. For more information, see the register reference manual at <https://ereg.caliptra.org>.
```

Comment:
`https://ereg.caliptra.org` URL doesn't work.

Reply:
it works for me

Reply:
@steph-morton will follow up internally (these are Google-owned redirects)